core/common/unreachable.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_MACRO_HPP
+#define KAGOME_MACRO_HPP
```

Comment:
```suggestion
#define KAGOME_UNREACHABLE_HPP
```
Or file name to `macro`, but I think better to change include guard